src/Cameras/Inputs/babylon.arcrotatecamera.input.pointers.ts

```diff
@@ -135,7 +143,9 @@ module BABYLON {
 
             this._onLostFocus = () => {
                 //this._keys = [];
-                pointers.empty();
+                pointers.clear();
+                pointersOrder = new Array<number>();

```

Comment:
I'm fixing the whole implem, but someone will have to check if the pinch still work (I don't have touch screen with me)